tst/testall.g

```diff
@@ -1,31 +1,11 @@
-TestMyPackage := function( pkgname )
-local pkgdir, testfiles, testresult, ff, fn;
-LoadPackage( pkgname );
-pkgdir := DirectoriesPackageLibrary( pkgname, "tst" );
+#
+# This file runs package tests. It is also referenced in the package
+# metadata in PackageInfo.g.
+#
+LoadPackage( "OpenMath" );
 
-# Arrange testfiles as required
-testfiles := [ "openmath02.tst", "openmath03.tst", "test_new" ];
+TestDirectory(DirectoriesPackageLibrary( "OpenMath", "tst" ),
+  rec(exitGAP := true));
```

Comment:
Please keep using `rec(compareFunction := "uptowhitespace")` like it was before. There are derived tests (which have to be placed under version control, and then they will be tested by Travis and wrapped by releaseTools - it is an oversight now, as you see  "openmath02.tst", "openmath03.tst" are not in the repository). Manual examples may have pretty formatting and fail if the comparison is exact. 

You can generated them (as part of building the manual) and put under version control - then Travis CI will also test them. Although, I have a better version of examples extraction script in the Congruence package https://github.com/gap-packages/congruence - I've done this and some other changes around tests there just today. Sstarted to give a "spring clean" to my packages in alphabetic order :)